lib/Feature/FeatureModelParser.cpp

```diff
@@ -1,6 +1,7 @@
 #include "vara/Feature/FeatureModelParser.h"
 
 #include "XmlConstants.h"
+#include "SxfmConstants.h"
```

Comment:
```suggestion

#include "llvm/ADT/StringExtras.h"
#include "llvm/Support/FormatVariadic.h"

#include "SxfmConstants.h"
#include "XmlConstants.h"
```
Fixes include ordering and adds includes needed for other suggestions further down.